packages/common-settings-connector/README.md

```diff
@@ -0,0 +1,6 @@
+# @twake/common-settings-connector
+The common settings connector module listens for new changes applied to the user profile and applies them for chat using
+the admin settings api.
```

Comment:
I agreed with @rezk2ll . We'll have a lot of communication (common settings, premium, deletion ...) that will use RabbitMQ. 

Since you're providing a wrapper around amqplib, either this wrapper should have its own package, either we rename this package to a more generic name with a domain dedicated to "common-settings".